[GLOSSARY] minor fixes to general bitcoin (#819)
zkSNACKs/WasabiDoc: docs/glossary/Glossary-GeneralBitcoin.md

### Transaction

In simple terms, a transfer of bitcoin.

> More precisely, a transaction is a signed data structure expressing a transfer of value from inputs to outputs.

Reviewer: I am not sure about this one.

Author: yeah, I was stumbling on this one too.
I just want to make clear that the main part of the tx is inputs and outputs. 

how about this

```suggestion
More precisely, a transaction is a signed data structure consisting of inputs and outputs.
```

Reviewer: I changed it to `More precisely, a transaction is a signed data structure expressing a transfer of value and consisting mainly of inputs and outputs.`

> More precisely, a transaction is a signed data structure expressing a transfer of value from inputs to outputs.

Reviewer: ```suggestion
More precisely, a transaction is a signed data structure expressing a transfer of value and consisting mainly of inputs and outputs
```

> More precisely, a transaction is a signed data structure expressing a transfer of value and consisting mainly of inputs and outputs

Reviewer: ```suggestion
More precisely, a transaction is a signed data structure expressing a transfer of value and consisting mainly of inputs and outputs.
```
